terraform/fullnode/gcp/variables.tf

```diff
@@ -41,6 +41,11 @@ variable "helm_force_update" {
   default     = false
 }
 
+variable "namespace" {
```

Comment:
can we name it "k8s_namespace" to be explicit, or else it might be easy to confuse with terraform workspace

Reply:
Yes, that makes sense.  I have updated the repo with the variable name change.